.github/PULL_REQUEST_TEMPLATE/release_template.md

```diff
@@ -0,0 +1,32 @@
+### Manual Release Steps
+
+## Features ⭐
```

Comment:
I recall a section "Tweaks" that is missing from this list. Perhaps there was a decision to consolidate tweaks into the features section, but it could also be that we just didn't use it in the PR you've taken this from

Reply:
You recall correctly! Though it looks like the last time we had a **Tweaks** section was around this time last year: #4267

Reply:
I like it, personally. Separates the wheat from the chaff.